modules/network-observability-flowmetrics-charts.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// network_observability/metrics-alerts-dashboards.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-custom-charts-flowmetrics_{context}"]
+= Configuring custom charts using FlowMetrics API
+You can generate charts for dashboards in the {product-title} web console, which you can view as an administrator in the *Dashboard* menu by defining the `charts` section of the `FlowMetric` resource.
+
+Here is an example for the `flowmetric-cluster-external-ingress-traffic` resource described above:
+
+.Generate a metric that tracks ingress bytes received from cluster external sources
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowMetric
+metadata:
+  name: flowmetric-cluster-external-ingress-traffic
+# ...
+  charts:
+  - dashboardName: Main
+    title: External ingress traffic
+    unit: Bps
+    type: SingleStat
+    queries:
+    - promQL: "sum(rate($METRIC[2m]))"
+      legend: ""
+  - dashboardName: Main
+    sectionName: External
+    title: Top external ingress traffic per workload
+    unit: Bps
+    type: StackArea
+    queries:
+    - promQL: "sum(rate($METRIC{DstK8S_Namespace!=\"\"}[2m])) by (DstK8S_Namespace, DstK8S_OwnerName)"
+      legend: "{{DstK8S_Namespace}} / {{DstK8S_OwnerName}}"
+# ...
+----
++
+This creates two panels:
+* A textual "single stat" showing the global external ingress rate summed across all dimensions
+* A timeseries graph showing the same metric per destination workload
```

Comment:
This isn't rendered correctly, cf screenshot:

![image](https://github.com/openshift/openshift-docs/assets/2153442/c5ccc88b-0297-4a3f-a369-1156ad6aa92d)